tests/virt/node/descheduler/conftest.py

```diff
@@ -106,15 +106,15 @@ def created_descheduler_subscription(
 
 @pytest.fixture(scope="module")
 def generated_descheduler_icsp_idms(
-    tmp_path_factory, generated_pulled_secret, openshift_current_version, ocp_qe_art_image_url, is_idms_cluster
+    pull_secret_directory, generated_pulled_secret, openshift_current_version, ocp_qe_art_image_url, is_idms_cluster
 ):
-    return create_folder_and_mirror_file(
-        path_factory=tmp_path_factory,
-        operator_name=DESCHEDULER_OPERATOR_DEPLOYMENT_NAME,
-        image=ocp_qe_art_image_url,
-        pull_secret=generated_pulled_secret,
-        is_idms_file=is_idms_cluster,
-        cnv_version=openshift_current_version,
+    return get_generated_icsp_idms(
+        image_url=ocp_qe_art_image_url,
+        registry_source=BREW_REGISTERY_SOURCE,
```

Comment:
How can it be brew source? https://github.com/RedHatQE/openshift-virtualization-tests/blob/main/tests/conftest.py#L2313

Reply:
The way `create_icsp_idms_command` is wrong, the command usage is: `oc adm catalog mirror SRC DEST [flags] [options]`

Since we are using `--manifests-only --to-manifests` there is no meaning to `source_url` which takes the DEST position.

For now I will put a placeholder string, but we need to change `create_icsp_idms_command` in another PR to represent better it's usage

Reply:
The current use of create_icsp_idms_command can be a bit misleading. The intended usage of the command is oc adm catalog mirror SRC DEST, but we’re using registry_source as DEST.

Since we include --manifests-only --to-manifests, the DEST field doesn’t actually serve a purpose. To make this clearer, I’ve replaced registry_source with a placeholder for now.

We should consider refactoring create_icsp_idms_command in a future PR to better align with its intended use.